### ISSUE TITLE
[CHANGE] vlt install to sort dependencies after adding new ones

### ISSUE THREAD

**Issue by KieranP** [vltpkg/vltpkg#238]
When using `vlt install` to add a new package, it currently adds it at the bottom.

Like most other package managers, the list should be sorted alphabetically.

**Comment by isaacs** [comment]
Agreed, we should always sort dependencies any time we write anything to `package.json`.
